src/chapters/aspdotnet-model-classes/data-layer.rst

```diff
@@ -0,0 +1,284 @@
+.. _data-layer:
+
+Models and Data 
+===============
+
+In order to work with data, we need to add another element to our MVC application. Say for example,
+we want to do things like remove an event from our list. Well, if two events both have the same name, 
+how might we identify which of those items to delete? We can't yet. So we need to tweak how we store
+event data. 
+
+In ``CodingEvents``, we add a unique identifier field to ``Events`` to better handle and track distinct 
+``Event`` instances. We'll also create another model class called ``EventData`` to encapsulate data storage 
+and prepare ourselves for decoupling data from controller classes.
```

Comment:
The first sentence very much ties to the example provided above, but as a student, I am not sure that I would understand based on the example why I need to create another model class.

Reply:
i changed this a bit but reminder that the previous two pages pretty strongly emphasize why want to do this and the example above is simply meant to supplement the notion of good data handling